backend/models/core/Entity.js

```diff
@@ -204,52 +204,112 @@ class Entity {
 
     return true;
   }
+
+  
+
+
     //Work in progress
-  static async update({entries, entity_type_id, entity_id}) {
-    
+  static async update({entries, entity_type_slug, entity_id}) {
+      
     const db = new DB();
-
-    //TODO: start transaction
     
+    // entity_type_slug will always be a string
+    // entity_id can be a string or a number
+   // check the entity_id weather it consists numbers only in which return true, otherwise it will return false
+
+    const {propertyType , conditionType} = isNumber(entity_id) ? 
+          {propertyType:'longValue',conditionType:'entities.id'} : 
```

Comment:
Apply proper spacing for readability.
```suggestion
          { propertyType: 'longValue', conditionType: 'entities.id' } : 
```